packages/jest-runtime/src/index.ts

```diff
@@ -723,6 +723,27 @@ export default class Runtime {
     return this.linkAndEvaluateModule(module);
   }
 
+  async unstable_importInternalModule(
```

Comment:
hmm, I wonder if we should say "if you use ESM, use TLA"? This doesn't seem worth it to me

Reply:
Can’t agree more. It is here because of https://github.com/facebook/jest/pull/12042#discussion_r743554657
(;

Reply:
Perhaps let’s land without this extra weight? Can be extended later if there is a need.

Reply:
yes, let's! can mention TLA in the docs, tho